docs/conf.py

```diff
@@ -0,0 +1,87 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+import sys
+import os
+from pathlib import Path
+import caf.distribute
+
+dir_path = Path(__file__).parents[1]
+source = dir_path / "src" / "caf" / "distribute"
+
+sys.path.insert(0, os.path.abspath(str(source)))
+sys.path.insert(
+    0,
+    os.path.abspath(r"C:\Users\IsaacScott\anaconda3\envs\distribute\Lib\site-packages"),
+)
```

Comment:
Is this path just used for local builds? 
We seem to be able to publish to ReadTheDocs, so I would guess so.

Can you make it relative so anyone can build local docs?